routes/pull_request.js

```diff
@@ -6,6 +6,22 @@ var thunkify = require('thunkify');
 
 module.exports = function(runtime) {
   return function * (detail) {
+    // Only actions that can introduce a bug number need to be handled. This
+    // also helps avoid race conditions in cases where multiple events are
+    // generated at the same time (for example when a PR has an assignee set
+    // at the time of creation, both "opened" and "assigned" events are sent).
+    // Possible actions are listed here:
+    // https://developer.github.com/v3/activity/events/types/#pullrequestevent
+    var relevantActions = {
```

Comment:
Yeah I'd originally wanted to use `.includes()` however that's only in ES7, which is not supported until node 6:
http://kangax.github.io/compat-table/esnext/#test-Proxy_internal_calls,_Array.prototype.includes

Autolander is currently using nodejs 4.2 on Heroku:
https://github.com/mozilla/autolander/blob/581655ecbeddad734bec86887f6b548694ed935f/package.json#L18

So in the meantime I was copying the pattern used here:
https://github.com/mozilla/autolander/blob/581655ecbeddad734bec86887f6b548694ed935f/lib/web.js#L28-L30

Open to suggestions :-)

(Separately I might try updating to nodejs 6, but things might start to break - we were lucky in #36 and since then our pinned dependencies/devDependencies are now likely even more behind latest)